doc/src/manual/workflow-tips.md

```diff
@@ -11,7 +11,7 @@ your experience at the command line.
 ### A basic editor/REPL workflow
 
 The most basic Julia workflows involve using a text editor in conjunction with the `julia` command
-line. A common pattern includes the following elements:
+line. To keep the code changes made by the editor in sync with the REPL session, there are multiple options. The package [Revise.jl](https://github.com/timholy/Revise.jl) can be used to manage this synchrony automatically. Alternatively, the following setup is recommended: 
```

Comment:
IMO it's the other way around: "keep the REPL session in sync with the code changes in the editor".